app/gas/management/commands/create_gas_history.py

```diff
@@ -0,0 +1,59 @@
+'''
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
```

Comment:
Multi-line spanning with `'''` can cause problems.  Moving forward, can we use `"""` to span lines?

Reply:
sure, but we should backport all the other views that have this style of top level comment in it.  this was pasted from another mgmt command